test/go-tests/test_webhook.go

```diff
@@ -404,28 +404,10 @@ spec:
         - url: http://shipyard-controller:8080/v1/project/{{.data.project}}/stage/{{.data.stage}}
           method: GET`
 
-func Test_Webhook_Alpha(t *testing.T) {
```

Comment:
Is there a functional reason for these changes in `test_webhook.go` ? Or is it just refactoring to make it nicer ?

Reply:
I am recycling the create and the webhook test in the zd tests 😄